arbitrary-file-write/file-create/README.md

```diff
@@ -49,4 +49,21 @@ asdas
 
 [we]! can have symbols and garbage at the end
 <00><ff> binary also ok here
-```
\ No newline at end of file
+```
+### Shell Profiles
+#### Login Shells
+`--login` flag. 
+* `/etc/profile` executed for all users
+* First found of `~/.bash_profile`, `~/.bash_login`, and `~/.profile` is then executed
```

Comment:
I guess these files might not exist. in that case it's relevant to this `file-create` section. However we are also missing examples, messy files, etc.